src/app/space/create/deployments/apps/deployments-apps.component.spec.ts

```diff
@@ -53,14 +61,22 @@ describe('DeploymentsAppsComponent', () => {
 
   initContext(DeploymentsAppsComponent, HostComponent,
     {
-      declarations: [FakeDeploymentCardContainerComponent, FakeDeploymentsToolbarComponent]
+      declarations: [FakeDeploymentCardContainerComponent, FakeDeploymentsToolbarComponent],
+      providers: [
+        { provide: Broadcaster, useValue: mockBroadcaster }
+      ]
     },
     (component: DeploymentsAppsComponent) => {
       component.spaceId = spaceId;
       component.environments = mockEnvironments;
       component.applications = mockApplications;
     });
 
+  beforeEach(() => {
+    fixture = TestBed.createComponent(DeploymentsAppsComponent);
+    component = fixture.debugElement.componentInstance;
+  });
```

Comment:
This is weird. Why do you need to create the component and get the component instance in this way? The rest of the tests in this spec use ex. `this.fixture.debugElement`, where `this` is the test context object. It seems like this should work fine for this use case as well.

Reply:
Ah I had forgotten to pass the context into the test so it wasn't working initially, I've made this revision.